api/orm/db_test.py

```diff
@@ -0,0 +1,26 @@
+""" Test to connect to database and execute transaction
+"""
+
+from db_init import SESSION, BASE, ENGINE
+from models.employee import Employee
+
+BASE.metadata.create_all(ENGINE)
+CURRENT_SESSION = SESSION()
+
+EMPLOYEE = Employee(idir="nkuruba",
+                    status="online",
+                    location="victoria",
+                    phone="123-456-7890")
+# create
+CURRENT_SESSION.add(EMPLOYEE)
+CURRENT_SESSION.commit()
+
+# Read
+EMPLOYEES = CURRENT_SESSION.query(Employee)
+for EMPLOYEE in EMPLOYEES:
+    print(EMPLOYEE.idir)
```

Comment:
I feel tests should be more robust. Any plans to add assertions?

Reply:
@Kyubinhan this test is just for demo purpose that it successfully connects to DB. However, this file is going to be removed in next pull request.

Reply:
@Kyubinhan this is not a unit test - and I've asked this be removed as part of https://github.com/bcgov/epr/issues/19